content/about-big-data.md

```diff
@@ -0,0 +1,159 @@
+
+### The Big Data promise
+The magic of Big Data is, supposedly, that you can get interesting and significant results using very simple methods and models, as long as the data volume is very large. The patterns just emerge. One example of this is the Google Trends graph on top of this post. We can make statements about the relative public interest in these subjects, based on the assumption that there is a relation between peoples' search behaviour and real-life events and trends. In this case we do not care about the causality of this relation. The method used is extremely simple: count the number of times a search occurs, yet the results are interesting because of the immense volume of the data.
+
+If you wonder about the relation between Google searches and real-life events, below is an example that is hard to deny. In the chart below, the dots on the blue line indicate the relative week-by-week volume of searches for the phrase "full moon". The weeks in red dots are weeks when a full moon occurred, the weeks in blue dots the remaining ones. There's a very clear relation between the search volume and the real-life event. It's easy for us to understand that the searches are not causing a full moon, just as the searches for Big Data are not causing the interest in the subject. For a model that predicts the next full moon (or wave of interest in a topic), however, this isn't relevant. You can just count words on the internet to make statements about the real world: Big Data at its finest.
+
+![full-moon](static/images/about-big-data/full-moon.png)
+<br/><small>Source: [Google Trends](http://www.google.com/trends/explore#q=full%20moon&date=today%2012-m&cmpt=q&tz=) and [moongiant.com](http://www.moongiant.com/Full_Moon_New_Moon_Calendar.php). Note: Google Trends only exposes week-by-week data, so the full moon dates are normalised to the start of the week they occured in as well.</small>
+
+As it turns out, while the above examples are nice, counting words on the internet doesn't solve everybody's business problems. We want to be able to make more granular predictions than those about the entire population of Google users. We care more specifically about our own customers or communities of users. These customers have come to expect a very high level of personlisation and customisation in our products, yet the data that we collect on these customers is typically very sparse. As a online retailer you may have millions of visitors, but each of those visitors individually might only visit your shop once or twice a year. Nonetheless, we are supposed to send them personlised e-mail campaigns, provide spot on product recommendations and be able to tell whether there is opportunity for up-sell or cross-sell. All of this based on those one or two visits per year.
```

Comment:
> We want to be able to make more granular predictions than those about the entire population of Google users. We care more specifically about our own customers or communities of users. These customers have come to expect a very high level of personlisation and customisation in our products, yet the data that we collect on these customers is typically very sparse.
→
> We want to predict more granularly than Google user base. We care about our own customers or communities of users. Because They have come to expect a very high level of personalisation and customisation in Our products. Yet the data we collect about them is typically very sparse